tests/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ FetchContent_Declare(
 
 FetchContent_MakeAvailable(googletest)
 
-add_executable(thyme_tests globals.cpp test_crc.cpp test_filesystem.cpp)
+add_executable(thyme_tests globals.cpp test_crc.cpp test_filesystem.cpp test_w3d_math.cpp)
```

Comment:
I feel you should break this out into a `set(TESTS_SRC)` maybe not in this PR but soon.

Reply:
Yes, i can do this in another PR